src/include/osl_pvt.h

```diff
@@ -44,9 +44,8 @@ class StructSpec;
 /// Kinds of shaders
 ///
 enum ShaderType {
```

Comment:
Did you mean to make this one an `enum class` as well?

Reply:
yes. why did  that work?